### PR TITLE
Fixed and added solution 2

### PR DESCRIPTION
**Pull request by spkulkarni33**
I had mistakenly not added the code for solution 2.

### CODE REVIEW

File: challenges/Solution 2/TestProject/src/restApi/RestApiCall.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+@Path("/rest")
+public class RestApiCall {
+
+	private static List<Integer> numberListInt;
+	private static final int expectedListSize = 500;
+
+	@POST
+	@Consumes(MediaType.MULTIPART_FORM_DATA)
+	@Path("/data")
+	public String validateNumbersPost(@FormDataParam("file") InputStream fileInputStream) {
```
Comment (sbrewer-headstorm) [inline challenges/Solution 2/TestProject/src/restApi/RestApiCall.java]: I know you mentioned you are new to rest APIs. As a suggestion in the future, consider using the Spring Rest Controller inside Java. It manages much of the API and data handling for you. Then you could just have an input of List<String> numberList and Spring would give automatically translate the JSON data into the Java object for you or return an error to client if it doesn't match.

File: challenges/Solution 2/TestProject/src/restApi/RestApiCall.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+		return new String("List accepted successfully");
+	}
+
+	@GET
+	@Produces(MediaType.TEXT_PLAIN)
+	@Path("/data")
+	public String sortNumbers() {
+
+		if (numberListInt == null)
+			return "List of numbers is empty";
```
Comment (sbrewer-headstorm) [inline challenges/Solution 2/TestProject/src/restApi/RestApiCall.java]: In HTTP there is a concept of HTTP Status Codes. In this situation you are returning an error message. What HTTP Status Code do you think you should send along with this message?

File: challenges/Solution 3/Solution3.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+		workphone = r["Work Phone"]
+		email = r["Email"]
+		address = r["Address"]
+		b_order = r["Basic Widget Order"]
+		aorder = r["Advanced Widget Order"]
+		pplan = r["Protection Plan"]
+
+		if email not in ids.keys():
+			ids[email] = i
+			print("INSERT INTO PERSON VALUES("+str(i)+", \""+str(name)+"\", \""+str(email)+"\", \""+str(address)+"\")")
```
Comment (sbrewer-headstorm) [inline challenges/Solution 3/Solution3.py]: Given your SQL, how would you plan to manage the relationship between your person table and phone tables? 
Are there any constraints you would enforce on fields in the new model?

File: challenges/Solution 2/TestProject/src/restApi/RestApiCall.java [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+		String jsonString = null;
+		String actualJsonString = null;
+		List<Integer> numberList = new ArrayList<>();
+		
+		try {
+			ByteArrayOutputStream result = new ByteArrayOutputStream();
+			byte[] buffer = new byte[1024];
+			int length;
+			while ((length = fileInputStream.read(buffer)) != -1) {
+				result.write(buffer, 0, length);
```
Comment (CharlesAHunt) [inline challenges/Solution 2/TestProject/src/restApi/RestApiCall.java]: Could you explain what you're doing with the `fileInputStream`? Are you storing the number data in a file on disk for permanent storage?

**Comment by CharlesAHunt** [comment]
Hey @spkulkarni33. Thanks for your PR! We have some questions for you in the form of PR comments that you can answer to move forward with out interview process. :+1:

**Pull request closed without merging**